internal/local/dependencies_test.go

```diff
@@ -100,7 +99,7 @@ func TestDependenciesPrepare(t *testing.T) {
 	testUpload, testUploadErr := os.Open(filepath.Join(wd, "testdata/dependencies/upload.zip"))
```

Comment:
it seems like this is meant to be the "transpiled" version? Not sure if there's a better way to rewrite the test to make it more meaningful. Again, depends on whether PrepareUpload() is even necessary.

Reply:
You can get rid of this test if you manage to remove the `PrepareUpload` function.